src/test/java/com/mathworks/ci/task/MatlabTestTaskTest.java

```diff
@@ -101,14 +105,18 @@ public void testExectuteRunsWithProvidedArguments() throws TaskException, IOExce
         configurationMap.put("srcFolderChecked", "true");
         configurationMap.put("byFolderChecked", "true");
         configurationMap.put("byTagChecked", "true");
+        configurationMap.put("strictChecked", "true");
+        configurationMap.put("useParallelChecked", "true");
         configurationMap.put("junit", "junit.xml");
         configurationMap.put("pdf", "report.pdf");
         configurationMap.put("html", "code-coverage");
         configurationMap.put("htmlModel", "model-coverage");
         configurationMap.put("stm", "results.mldatx");
         configurationMap.put("srcfolder", "src/:src1");
         configurationMap.put("testFolders", "test/");
-        configurationMap.put("testTag", "all");      
+        configurationMap.put("testTag", "all");
+        configurationMap.put("outputDetail", "Detailed");
+        configurationMap.put("loggingLevel", "Detailed");
```

Comment:
Are we picking Detailed as the default level of verbosity? Is this consistent with Jenkins ? Do you know why?

Reply:
- Here `Detailed` is just an input for testing which is later verified in line 141-142.
- The default level of verbosity would be same as the arguments' actual default (`Default` keyword is used for this purpose, similar to Jenkins).